SampleGeneration/20250518_ForestReducer_RAAOxygenOxygen/ReduceForest.cpp

```diff
@@ -272,6 +273,24 @@ int main(int argc, char *argv[]) {
         }
       }
 
+      ////////////////////////////////////////
+      ///// Fill default selection bits //////
+      ////////////////////////////////////////
+
+      if (IsPP) {
+        // If PP sample
+        MChargedHadronRAA.passBaselineEventSelection = getBaselinePPEventSel(MChargedHadronRAA);
+        // FIXME: Check if the HF information is present in pp
+      } else {
+        // If OO sample
+        MChargedHadronRAA.passBaselineEventSelection = getBaselineOOEventSel(MChargedHadronRAA);
+        // Fill HF selection bits
+        MChargedHadronRAA.passHFAND_6p06p0_Offline = checkHFCondition(MChargedHadronRAA, 6., 6., false);
+        MChargedHadronRAA.passHFAND_6p06p0_Online = checkHFCondition(MChargedHadronRAA, 6., 6., true);
+        MChargedHadronRAA.passHFOR_8p0_Offline = checkHFCondition(MChargedHadronRAA, 8., -2., false);
```

Comment:
Using a specific value (-2) to mimic a HFOR condition could generate problems in the future, I fear. I would rather implement a separate OR and AND condition.